third_party/terraform/utils/config.go.erb

```diff
@@ -70,6 +70,7 @@ type Config struct {
 	Scopes         		[]string
 	BatchingConfig		*batchingConfig
 	UserProjectOverride bool
+    SynchronousTimeout  time.Duration
```

Comment:
I would lean towards "RequestTimeout" or "HTTPTimeout" as this is going to affect every request that Terraform makes to the API including polling of async resources.